catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/converter/mgmp/MgmpConverter.java

```diff
@@ -0,0 +1,1230 @@
+
+    private void addMetadataConstraints(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        addMetadataSercurityClassification(pathValueTracker, metacard);
+        addOriginatorSecurity(pathValueTracker, metacard);
+        addMetadataReleasability(pathValueTracker, metacard);
+    }
+
+    private void addDistributionInfo(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        addGmdMetacardFormat(pathValueTracker, metacard);
+        addGmdResourceUri(pathValueTracker, metacard);
+    }
+
+    @SuppressWarnings("UnusedParameters")
+    private void addContentInfo(XstreamPathValueTracker pathValueTracker, MetacardImpl metacard) {
+        addImageDescription(pathValueTracker, metacard);
+    }
+
+    private void addMdIdentification(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        addMdIdentificationCitation(pathValueTracker, metacard);
+        addMdIdentificationAbstract(pathValueTracker, metacard);
+        addMdIdentificationStatus(pathValueTracker, metacard);
+        addMdIdentificationPointOfContact(pathValueTracker, metacard);
+        addMdIdentificationDescriptiveKeywords(pathValueTracker, metacard);
+        addMdIdentificationResourceConstraints(pathValueTracker, metacard);
+        addMdIdentificationAggregationInfo(pathValueTracker, metacard);
+        addMdIdentificationLanguage(pathValueTracker, metacard);
+        addMdIdentificationTopicCategories(pathValueTracker, metacard);
+        addMdIdentificationExtent(pathValueTracker, metacard);
+    }
+
+    private void addMdIdentificationExtent(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        addGeographicIdentifier(pathValueTracker, metacard);
+        addBoundingPolygon(pathValueTracker, metacard);
+        addTemporalElements(pathValueTracker, metacard);
+        addVerticalElement(pathValueTracker, metacard);
+    }
+
+    private void addMdIdentificationResourceConstraints(XstreamPathValueTracker pathValueTracker,
+            MetacardImpl metacard) {
+        addResourceSercurityClassification(pathValueTracker, metacard);
+        addMdIdentificationResourceConstraintsOriginatorClassification(pathValueTracker, metacard);
+        addMdIdentificationResourceConstraintsCaveats(pathValueTracker, metacard);
+    }
+
+    private void addMdIdentificationResourceConstraintsOriginatorClassification(
+            XstreamPathValueTracker pathValueTracker, MetacardImpl metacard) {
+        Utilities.addFieldIfString(pathValueTracker,
+                metacard,
+                Security.RESOURCE_ORIGINATOR_CLASSIFICATION,
+                MgmpConstants.RESOURCE_ORIGINATOR_SECURITY_PATH,
+                (pathValueTracker1, metacard1) -> {
+                    pathValueTracker1.add(new Path(RESOURCE_ORIGINATOR_SECURITY_CODE_LIST_PATH),
+                            "http://mod.uk/spatial/codelist/mgmp/2.0/MGMP_ClassificationCode");
```

Comment:
Recommend making this a constant to differentiate this from the `CLASSIFICATION_CODE` since they are very similar.